browser/css/menubar.css

```diff
@@ -63,14 +63,60 @@
 	display: none;
 }
 .main-nav {
-	height: 32px; /* on mouseover menubar items, border emerges */
+	height: 36px; /* on mouseover menubar items, border emerges */
 	width: auto;
 	background: #ffffff;
-	padding: 3px 3px 3px 0;
+	padding: 0;
 	white-space: nowrap;
 	z-index: 12;
 	display: flex;
+	align-items: center;
```

Comment:
Vertical align nav bar and its child: maybe better to have a commit only fr this

Reply:
_cherry-picking and creating a new commit just with your Vertical align nav bar changes_

Reply:
Tested and the lack of padding made it look very cramped up (without enough white space around), pick everything else though: https://github.com/CollaboraOnline/online/pull/3952